style/battle-log.css

```diff
@@ -163,6 +163,9 @@ button:disabled {
 	text-shadow: none;
 	color: #F9F9F9;
 }
+.dark select.button {
+	background-color: #2b2c31 !important;
```

Comment:
I have a guess as to what's going on here but what browser is this bad???

Reply:
I use chrome

Reply:
Hah, it turns out that Chrome for Windows has this problem, but Chrome for Mac doesn't.

Anyway, interesting solution, but it turns out styling `option` works, and doesn't require your hack that makes select buttons uglier.